ui/src/timeMachine/actions/index.ts

```diff
@@ -36,7 +36,7 @@ import {HistogramPosition} from '@influxdata/giraffe'
 import {RemoteDataState} from '@influxdata/clockface'
 import {createView} from 'src/shared/utils/view'
 import {setValues} from 'src/variables/actions'
-import {getTimeRangeByDashboardID} from 'src/dashboards/selectors'
+import {getTimeRangeByDashboardID} from 'src/dashboards/selectors/index'
```

Comment:
why did you have to be so specific here?

Reply:
better to be explicit so that there's no doubt about what the function does at a glance